src/lib.rs

```diff
@@ -131,6 +131,8 @@
 #![cfg_attr(docsrs, feature(doc_auto_cfg, doc_cfg_hide), doc(cfg_hide(doc)))]
 #![deny(unsafe_op_in_unsafe_fn)]
 #![warn(clippy::undocumented_unsafe_blocks)]
+// Update in Cargo.toml as well.
+#![doc(html_root_url = "https://docs.rs/raw-window-metal/1.0.0")]
```

Comment:
I think it's more frequent/prone to update `Cargo.toml` while forgetting to update this. Also seems like this should be the default when building with `--no-deps` (and the crate source comes from `crates.io` with a possibly-defaulted `doucmentation =` link)?

Reply:
Yeah, and that while `raw-window-handle` explicitly lists the URL instead of relying on [the default URL](https://doc.rust-lang.org/cargo/reference/manifest.html#the-documentation-field) (when being packaged).